spec/models/boosted_content_spec.rb

```diff
@@ -66,16 +74,6 @@
       expect(boosted_content.errors.full_messages.join).to match(/Publish end date can't be before publish start date/)
     end
 
-    it 'should not allow duplicate url' do
-      url = 'http://search.gov/post/9866782725/did-you-mean-roes-or-rose'
-      BoostedContent.create!(valid_attributes.merge(url: url))
-      expect { BoostedContent.create!(valid_attributes.merge(url: url)) }.to raise_error
-    end
-
-    it 'should not allow blank description' do
-      expect { BoostedContent.create!(valid_attributes.merge(description: '&nbsp;')) }.to raise_error
-    end
```

Comment:
Do we cover this elsewhere? Or is this no longer needed?

Reply:
Whoops - I meant to call this out. I replaced the duplication spec with a Shoulda spec (the new `is_expected.to validate_uniqueness_of...` spec). That old spec just jumped out at me for refactoring, because I prefer the Shoulda specs for readability and consistency.

The `should not allow blank description` spec was a bit of a misnomer, because it was testing two pieces of behavior: 1) that a description such as '&nbsp;' would be sanitized to `""`, and 2) the resulting empty string would be invalid. We have an existing spec for the presence validation (`it { is_expected.to validate_presence_of :description }`), and I added the new, separate spec to test the sanitization: `it_behaves_like 'a record that sanitizes attributes', [:title, :description]`.